### ISSUE TITLE
Train logs

### ISSUE THREAD

**Issue by mtalhabalci** [BreezeWhite/oemer#73]
Hey,

I've made a train with the first dataset for stafflines.

For proofing the concept i trained it with low epoch and step.

But i didn't manage to see a log mechanism here to see how the train went.

Before going further i want the first train to be fulfilled.

Am i missing a point or is there a structure in terms of loggingto see how the process went?

Greetings.

**Comment by liebharc** [comment]
Hi, I haven't trained this in a while. You should see a validation loss at the end of each epoch. So if you train for e.g. 3 epochs you should see that the loss decreases. A decreasing loss would be a good indication, but also nothing more than an indication. I don't remember exactly, but I think a full training run of unet and segnet took about 8h on my system. So it's something which can be done in a night - which is quick compared to other models.

As you asked for the rests_above model (and the other classifiers) over in the homr repo: I think the code which creates the datasets for the classifiers is also located in https://github.com/BreezeWhite/oemer/blob/main/oemer/classifier.py
